python/python/glide/async_commands/sorted_set.py

```diff
@@ -0,0 +1,154 @@
+# Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+from enum import Enum
+from typing import List, Optional, Union
+
+
+class InfBound(Enum):
+    """
+    Enumeration representing numeric and lexicographic positive and negative infinity bounds for sorted set scores.
+    """
+
+    POS_INF = {"default_arg": "+inf", "lex_arg": "+"}
```

Comment:
If +inf is only used with score, change default_arg to score_arg

Reply:
Have you checked if this API is possible in nodeJS, Java? (having one enum with two potential arg values)

Reply:
If it isn't, we might better choosing your prev solution